src/assets/illustrations/fallback/fallback_code.svg

```diff
@@ -0,0 +1,81 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="240" height="156" viewBox="0 0 190 106">
+  <defs>
+    <mask id="a" x="1.5" y="0" width="212.37799" height="112.4" maskUnits="userSpaceOnUse">
+      <rect x="1.5" width="187.8" height="106" rx="2" fill="#fff"/>
+    </mask>
+  </defs>
```

Comment:
Can we find a way to make this work without a mask? I don't see anything in the image that would require one, but I may be missing something.  (This feedback applies to all images with masks.)

Reply:
In Firefox, the masks are causing rendering issues for some images when displayed in an iframe:

<img width="780" alt="Screen Shot 2021-04-30 at 10 07 19 AM" src="https://user-images.githubusercontent.com/69633/116729313-deec8680-a99b-11eb-9280-af38619b0d45.png">